addons/point_of_sale/static/src/js/models.js

```diff
@@ -1214,11 +1214,16 @@ class PosGlobalState extends PosModel {
             }
 
             if(company.country && company.country.code === "IN"){
+                let total_tax_amount = 0.0;
                 for(const [i, tax_factor] of incl_tax_amounts.percent_taxes){
                     const tax_amount = round_pr(base_amount * tax_factor / (100 + percent_amount), currency_rounding);
+                    total_tax_amount += tax_amount;
                     cached_tax_amounts[i] = tax_amount;
                     fixed_amount += tax_amount;
                 }
+                for(const [i,] of incl_tax_amounts.percent_taxes){
```

Comment:
@smetl I removed the unused _tax_factor that was making the CI fail. I retested the PoS part and it's working fine